flexmeasures/ui/templates/crud/users.html

```diff
@@ -34,49 +50,133 @@ <h3>All {% if not include_inactive %}active {% endif %}users</h3>
                 <th class="d-none">URL</th>
               </tr>
             </thead>
-            <tbody>
-              {% for user in users %}
-              <tr>
-                <td>
-                    {{ user.username }}
-                </td>
-                <td>
-                    <a href="mailto:{{ user.email }}" title="Mail this user">{{ user.email }}</a>
-                </td>
-                <td>
-                    {% for role in user.flexmeasures_roles %}
-                    {{ role.name }}{{ "," if not loop.last }}
-                    {% endfor %}
-                </td>
-                <td>
-                    <a href="/accounts/{{user.account.id}}" title="View accounts">{{ user.account.name }}</a>
-                </td>
-                <td>
-                    {{ user.timezone }}
-                </td>
-                <td title="{{  user.last_login_at | localized_datetime }}" data-sort="{{ user.last_login_at }}">
-                    {{ user.last_login_at | naturalized_datetime}}
-                </td>
-                <td title="{{  user.last_seen_at | localized_datetime }}" data-sort="{{ user.last_seen_at }}">
-                    {{ user.last_seen_at | naturalized_datetime}}
-                </td>
-                <td>
-                    {{ user.active }}
-                </td>
-                <td class="d-none">
-                    /users/{{ user.id }}
-                </td>
-              </tr>
-              {% endfor %}
+            <tbody id="users-table-body">
+              <!-- Dynamic rows will be rendered here -->
             </tbody>
           </table>
         </div>
+        <nav aria-label="Page navigation example">
```

Comment:
Why "example"?

Reply:
I'll remove that, it came with the code snippet form Bootstrap I believe.